src/installer/test/Directory.Build.props

```diff
@@ -13,11 +13,11 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <TargetOSTrait Condition="'$(OSGroup)' == 'Windows_NT'">nonwindowstests</TargetOSTrait>
-    <TargetOSTrait Condition="'$(OSGroup)' == 'Linux'">nonlinuxtests</TargetOSTrait>
-    <TargetOSTrait Condition="'$(OSGroup)' == 'OSX'">nonosxtests</TargetOSTrait>
-    <TargetOSTrait Condition="'$(OSGroup)' == 'FreeBSD'">nonfreebsdtests</TargetOSTrait>
-    <TargetOSTrait Condition="'$(OSGroup)' == 'NetBSD'">nonnetbsdtests</TargetOSTrait>
+    <TargetOSTrait Condition="'$(BuildOS)' == 'Windows_NT'">nonwindowstests</TargetOSTrait>
+    <TargetOSTrait Condition="'$(BuildOS)' == 'Linux'">nonlinuxtests</TargetOSTrait>
+    <TargetOSTrait Condition="'$(BuildOS)' == 'OSX'">nonosxtests</TargetOSTrait>
+    <TargetOSTrait Condition="'$(BuildOS)' == 'FreeBSD'">nonfreebsdtests</TargetOSTrait>
+    <TargetOSTrait Condition="'$(BuildOS)' == 'NetBSD'">nonnetbsdtests</TargetOSTrait>
```

Comment:
It maybe out of scope for your PR, but this block was added in core-setup days to support `[PlatformSpecific(..)]` attribute in tests. It would be nice to remove this `PropertyGroup` and resolve TestAdditionalArguments` from https://github.com/dotnet/runtime/blob/c2d89a8/eng/testing/xunit/xunit.targets, which has the same logic (currently only used by libraries). https://github.com/dotnet/arcade/issues/4105

Reply:
@am11 currently we have a lot of duplicated properties among the three subsets. The final goal here is to have all the common properties defined at one place only. Feel free to take up the workitem for this set of properties :)

Reply:
@am11 can you please open an issue in dotnet/runtime to track removing the duplicate testing infrastructure?